src/Epr.Reprocessor.Exporter.UI/Program.cs

```diff
@@ -48,6 +50,8 @@
     options.AllowedHosts = forwardedHeadersOptions.AllowedHosts;
 });
 
+services.Configure<HomeSettings>(builderConfig.GetSection("Links"));

```

Comment:
Can you rename Homesettings to externalLinks there can be additional in future and they may not be specific to home?